demos/database.php

```diff
@@ -2,8 +2,8 @@
 
 // A very basic file that sets up Agile Data to be used in some demonstrations
 try {
-    if (file_exists('db.php')) {
-        include_once __DIR__ . '/db.php';
+    if (file_exists(dirname(__DIR__) . '/demos/db.php')) {
```

Comment:
We are already in demos/, no dirname needed :)

Reply:
I thought it was safer this way. I will change it.

Reply:
done!